documentation/synthetic_tokens/security/security.md

```diff
@@ -0,0 +1,107 @@
+* Solution: Bots should be performant enough that they can capture the  maximum profit from liquidation before the price moves too far. 
+Additionally, UMA will liquidate token sponsors uneconomically early on to support the development of synthetic tokens. 
+In a large market, liquidations should be effected immediately.
+
+### Dilution of liquidation rewards to miners
+* Scenario: The first person to call liquidate or dispute gets the reward, which results in a gas price war. 
+In the limit, the gas price for a liquidation or dispute will be as high as the reward itself, effectively sending the reward to the miner (“miner extractable value”).
+* Solution: This is a common problem for all DeFi “arbitrage” transactions. 
+One alternative is to use an auction for liquidation to move from a gas bidding war to a price bidding war for the reward.
+
+# Open Issues
+Below is a list of known open issues regarding the capital efficiency of token sponsor, liquidator, and disputer positions related to this financial contract template. 
+We’ve categorized and ranked the open issues by likelihood of impact. 
+
+Note that the financial contract template has an emergency shutdown feature that prematurely expires the contract. 
+This can only be triggered by a vote approved by UMA token holders. 
+Emergency shutdown prematurely expires the contract and the final resolution depends on a DVM price, which could take 2-4 days to arrive.
+
+## Open issues affecting incentives to liquidate:
+### No haircut mechanisms implemented
+* Scenario: If any token sponsor’s position is collateralized by < 100%, there is no incentive to liquidate them. 
+If the contract expires with any sponsors below 100% collateral, there will be a race to pull collateral from the contract. 
+Whichever sponsors and token holders redeem first will receive collateral. 
+Whoever comes last will receive less or nothing depending on the amount of collateral left in the contract.
+* Solution: Although we expect the economic incentives to work without a haircut mechanism, if a position does reach < 100% collateralization, UMA will liquidate these positions uneconomically.
+
+## Open issues affecting capital efficiency: 
+### New token creation
+* Scenario: Token sponsors who wish to create new tokens are required to collateralize the new tokens at levels that meet or exceed the global collateralization ratio (GCR). 
+However, this makes the role of token sponsor capital inefficient for 2 reasons: 
+1. If a token sponsor has an outstanding synthetic token position collateralized well above the GCR, such that the creation of new tokens could be supported by the current amount of excess collateral in their position, they cannot use this collateral to create new synthetic tokens. 
+1. If a token sponsor wishes to create new tokens collateralized to a level below the GCR, they must first collateralize their position above the GCR and go through a “slow” withdrawal to withdraw their excess collateral.
+
```

Comment:
What's the solution? We need to say something about how we address this. (The OZ doc does this).

Reply:
The OZ doc does not seem to have a solution, instead stating: 

Conclusion: Ignore this problem because capital efficiency isn’t a huge concern.

The lack of a solution is why this is listed as an "open issue", rather than an "attack vector" that has a proposed solution. 

Given that we seem to be removing the distinction between these, do you think we should state a "conclusion", as above? I think that would be an appropriate section under each issue that's raised.

Reply:
I like how you did this.